frontend/src/features/mapSideBar/tabs/ltsa/LtsaDuplicateTitleSubForm.tsx

```diff
@@ -18,7 +17,6 @@ export const LtsaDuplicateTitleSubForm: React.FunctionComponent<ILtsaDuplicateTi
     getIn(values, withNameSpace(nameSpace, 'duplicateCertificatesOfTitle')) ?? [];
   return (
     <>
-      <StyledSectionHeader>Duplicate Indefeasible Title</StyledSectionHeader>
```

Comment:
Yeah, I must be missing something here.

Reply:
This standardizes the different subforms to use the Section component instead of the styled version of it.